### PR TITLE
Add FEniCS 2023 page

### PR DESCRIPTION
**Pull request by francesco-ballarin**

### CODE REVIEW

**Comment by francesco-ballarin** [comment]
preview of the conference page
[FEniCS 2023 _ FEniCS Project.pdf](https://github.com/FEniCS/web/files/10267177/FEniCS.2023._.FEniCS.Project.pdf)

File: conference/2023.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+[code of conduct](../community/code-of-conduct.md). Instances of
+abusive, harassing, or otherwise unacceptable behavior may be reported
+by contacting the conference organizers or the [FEniCS Steering
+Council](https://github.com/FEniCS/governance).
+
+## Organizing committee
+
+- [Drew F. Parsons](https://www.unica.it/unica/it/ateneo_s07_ss01.page?contentId=SHD273808), [Università di Cagliari](https://www.unica.it/), Italy (chair)
+- [Francesco Ballarin](https://www.francescoballarin.it/), [Università Cattolica del Sacro Cuore](https://www.unicatt.it/), Brescia, Italy
+- ‪[Chris Richardson](https://www.esc.cam.ac.uk/directory/chris-richardson)‬, [University of Cambridge](https://www.cam.ac.uk/), United Kingdom
```
Comment (mscroggs) [inline conference/2023.md:100]:
```suggestion
- [Chris Richardson](https://www.esc.cam.ac.uk/directory/chris-richardson), [University of Cambridge](https://www.cam.ac.uk/), United Kingdom
```

**Comment by mscroggs** [comment]
Looks good to me, I say we go ahead and merge it

**Comment by francesco-ballarin** [comment]
> Looks good to me, I say we go ahead and merge it.

I'd wait for @chrisrichardson and @drew-parsons , since their names are mentioned on the web page ;)
In the meantime I'll move the PR out of draft state.

**Comment by jhale** [comment]
You can mention the travel wards but we haven't applied to NumFOCUS yet through their small grant awards., so maybe best to remove that.

**Review by drew-parsons — APPROVED** [review]
Nice image of Pula.  The chemists had a stylised version with flamingos added, I've asked if they're happy for us to reuse.

The draft looks good. Good to announce the dates so people can commence forward planning.

**Review by chrisrichardson — APPROVED** [review]

**Pull request merged**